src/main/java/studio/magemonkey/fusion/cfg/hooks/divinity/DivinityModule.java

```diff
@@ -0,0 +1,20 @@
+package studio.magemonkey.fusion.cfg.hooks.divinity;
+
+public enum DivinityModule {
+    Arrows,
```

Comment:
Convention for enums is all caps with underscores separating words. `ARROWS`, `CUSTOM_ITEMS`, etc. Unless there is a reason for tihs UpperCamelCase, I'd prefer we follow convention.

Reply:
Thats just a bad habit of mine for preference reasons. I have rechecked it and it turns out that I dont even had a real use for it. So i also removed the class for now